bitnami/airflow/templates/config/configmap-dags.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Files.Glob "files/dags/*.py" }}
```

Comment:
AFAIK we want to stop offering support for including files in the `files/` directory since it requires the chart to be cloned. Instead we want to use configmaps/secrets for this purpose. I think Elasticsearch had some examples of this. Could you revisit this?

Reply:
done!